pubspec.yaml

```diff
@@ -32,6 +32,8 @@ dependencies:
   provider: ^4.3.2
   adaptive_breakpoints: ^0.0.2
 
+  web_benchmarks: ^0.0.2
```

Comment:
⇒ dev_dependencies

Reply:
`web_benchmarks` is used by `lib/benchmarks/` as well, so it needs to be a regular dependency. [Reference](https://dart.dev/tools/pub/dependencies#dev-dependencies)

Reply:
Stepping back, do the benchmarks need to be in `lib`? They're effectively tests, so if we can, we should avoid including them in builds.

Reply:
This is a good idea. I'm testing it out.

Reply:
Done. It's looking great!